.github/workflows/plugin-verification.yml

```diff
@@ -0,0 +1,54 @@
+name: Plugin Verification
+description: |
+  This workflow is used to verify the plugin against different versions of IntelliJ IDEA.
+  It reads the minimum and maximum supported versions from the `gradle.properties` file
+  and runs the plugin against each version in the range.
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Runs every day at midnight UTC
```

Comment:
Do we need to run this on every night? 

Ideally, if the plugin works for a given IDEA version, it shouldn't break overnight, right?
In that case, I think the better strategy to run this on push, rather than run on schedule.

Reply:
yes makes sense. Updated with https://github.com/ballerina-platform/plugin-intellij/pull/73/commits/fee000d6015eb7c23257ba0e394bfefce1cd0981